### PR TITLE
Trying to make checks pass

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

**Comment by sbrunner** [comment]
Can you rebase on branch 2.3?

File: vars.yaml [hunk 1]
```diff
@@ -89,6 +89,7 @@ vars:
       constants:
         <<: *constants
         gmfTreeManagerModeFlush: false
+        # FIXME: gmfSearchActions: []
```
Comment (sbrunner) [inline vars.yaml]: Should be removed?

Comment (fredj) [reply vars.yaml]: yes, removed

**Review by sbrunner — APPROVED** [review]
Merge and backport it to c2cgeoportal :-)
Don't take care on Jenkins tests, I just remove the Jenkinsfile :-)

**Pull request merged**